.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,86 @@
+name: Prepare Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: "Release version tag (e.g. v1.2.3)"
+        required: true
+      ref:
+        type: string
+        description: "Git ref from which to release"
+        required: true
+        default: "main"
+      do_publish_jars:
+        type: boolean
+        description: "Publish Java artifacts for staging"
+        required: true
+        default: "true"
+env:
+  NEXUS_SDK_REF: ${{ github.event.inputs.ref }}
+  NEXUS_SDK_TAG: ${{ github.event.inputs.tag }}
+
+jobs:
+  create_draft_release:
+    name: Create Github draft release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Audit gh version
+        run: gh --version
+
+      - name: Check for existing release
+        id: check_release
+        run: |
+          echo "::echo::on"
+          gh release view --repo "$GITHUB_REPOSITORY" "$NEXUS_SDK_TAG" \
+            && echo "::set-output name=already_exists::true" \
+            || echo "::set-output name=already_exists::false"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout repo
+        if: steps.check_release.outputs.already_exists == 'false'
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.NEXUS_SDK_REF }}
+
+      - name: Create release
+        if: steps.check_release.outputs.already_exists == 'false'
+        run: >
+          gh release create
+          "$NEXUS_SDK_TAG"
+          --draft
+          --repo "$GITHUB_REPOSITORY"
+          --title "$NEXUS_SDK_TAG"
+          --target "$NEXUS_SDK_REF"
+          --generate-notes
```

Comment:
Missing `\` for the command line break

Reply:
Hm, I see. I like to keep things consistent, so I'd suggest changing the `>` to `|` as in other steps above, but a big deal either way.